docs/DeveloperGuide.md

```diff
@@ -624,3 +624,20 @@ testers are expected to do more *exploratory* testing.
+5. **Accept multiple social media handles**: Currently, there can be only 0 or 1 `SOCIALMEDIAHANDLE` stored for a given contact, and should an `add` or `edit` command with more than one `s\SOCIALMEDIAHANDLE` field be entered, only the last one would be stored. Some users may prefer to store multiple `SOCIALMEDIAHANDLE`s of their contacts, perhaps if the contact has different handles for different platforms, or different purposes. Hence, we plan to store `SOCIALMEDIAHANDLE` in a set so that contacts can have more than 1 `SOCIALMEDIAHANDLE`.
+6. **Accept multiple pieces of financial information**: Currently, there can be only 0 or 1 `FINANCIALINFO` stored for a given contact, and should an `add` or `edit` command with more than one `fi\FINANCIALINFORMATION` field be entered, only the last one would be stored. Some users may prefer to store multiple pieces of `FINANCIALINFO` of their contacts for different purposes. Hence, we plan to store `FINANCIALINFO` in a set so that contacts can have more than 1 piece of `FINANCIALINFO`.
+7. **Add horizontal scroll bar to UI to prevent text truncation**: Currently, if a user enters a very long value for certain fields, such as, but not limited to, `NAME` or `ADDRESS`, and the application is open in a window that is not wide enough to display the entire value of that field, the value will be truncated with `...` at the end. This could cause inconvenience to certain users, who may not be able to view the entire value as a result. Thus, we plan to modify the UI to create a horizontal scroll bar so that the user can see the entire value instead of it being truncated.
+8. **Accept negative numbers and mathematical expressions in Tag names and values**: Currently, the `TAG` and `TAG:value`'s TagName and TagValue fields do not accept mathematical symbols or expressions, including, but not limited to, `+`, `-`, `*` or `/`, along with negative numbers. This could cause an inconvenience to certain users, who may wish to store `TAG`s with such information. Hence, we intend to adjust the validation RegEx of the `Tag` class to allow for such expressions.
+9. **Block fullscreen of application in Unix and Unix-Like environments**: On certain systems with Unix-based or Unix-like operating systems, including but not limited to MacOS, Linux and its distributions, making the application run in full screen mode leads to slightly buggy behaviour in the GUI, such as the Help window not scaling fully to the size of the display. In This may inconvenience a minority of users who attempt to run the application in fullscreen mode. Hence, we intend to modify the `UiManager` file to block attempts for Unix-based and Unix-like operating systems from making the application run in fullscreen mode.
```

Comment:
"In This may inconvenience..." should not have "In" at the start I believe

Reply:
Unless I am mistaken, I also don't think we should say "minority of users" considering running applications in fullscreen is quite the norm.

We can instead just say "users who attempt to run...".